### PR TITLE
arm: dts/revpi-con-can: Remove pincontrol references for can0 [REVPI-1921]

### PR DESCRIPTION
**Pull request by nbuchwitz**
Remove pincontrol reference for <&conbridge_int_pins> in order to
restore the con can module functionality, which worked before these
changes. conbridge_int_pins is defined in revpi-connect-overlay.dts and
therefore not exported to the symbols table. This results in the
following error when the overlay is loaded:

OF: resolver: node label 'conbridge_int_pins' not found in live
devicetree symbols table

This partialy reverts commit 164a3d73bae2a

Signed-off-by: Nicolai Buchwitz <n.buchwitz@kunbus.com>

### CODE REVIEW

**Comment by iluminat23** [comment]
This will leave the pins unconfigured. It might work because the gpio controller of the bcm is totally broken. But we should do this right.
I see two options:

1. Move the pins to the GPIO/Pinctrl (not my preferred option)
2. Give the con-can overlay its own pinctrl

File: arch/arm/boot/dts/overlays/revpi-con-can-overlay.dts [hunk 1]
```diff
@@ -49,4 +49,15 @@
 			};
 		};
 	};
+
+	fragment@3 {
```
Comment (iluminat23) [inline arch/arm/boot/dts/overlays/revpi-con-can-overlay.dts]: I would put this as fragment@2 and move the spi/can to fragment@3

**Review by iluminat23 — CHANGES_REQUESTED** [review]
In the commit message title:
- The prefix is either _arm/dts/revpi-con-can:_ or _arm: dts: revpi-con-can:_ or you might skip the arm and only use _dts/revpi-con-can:_
- You are not removing the pincontrol reference. You are moving it

The 2nd line of the commit message is way to long.

**Comment by nbuchwitz** [comment]
@iluminat23 I updated the commit message but forgot about the title. Now it should be fixed. I also switched fragment 2 and 3

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**